create/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -20,7 +20,7 @@ After you create the pipelines for your application, you can view and visually i
 == Prerequisites
```

Comment:
@mramendi Why is the prereqs discrete here? I do not see such usage in the OCP docs or a mention such as this for prereqs in the OCP docs guidelines anywhere. Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Again not in scope as this is pre-existing content. Changing this would require a full review cycle